source/packages/services/assetlibrary/src/config/.env.defaults

```diff
@@ -23,7 +23,7 @@ CACHE_TYPES_TTL=30
 
 SUPPORTED_API_VERSIONS=application/vnd.aws-cdf-v1.0+json,application/vnd.aws-cdf-v2.0+json
 
-AUTHORIZATION_ENABLED=true
```

Comment:
unrelated change to temporarily resolve https://github.com/aws/aws-connected-device-framework/issues/131